### ISSUE TITLE
quiet describe in tslint cli options

### ISSUE THREAD

**Issue by dawncold** [palantir/tslint#4687]
### Bug Report

- __TSLint version__: N/A
- __TypeScript version__: N/A
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
// code snippet
```

with `tslint.json` configuration:

```json

```

#### Actual behavior
```console
 -q, --quiet                            hide errors on lint
```

#### Expected behavior
```console
 -q, --quiet                            hide warnings on lint
```
